asylum-connect/catalog-api
# Feature/1198892993658136/sort by distance

## Description

- adds geoNear query to support sorting search results by distance when searching by location
- updates tests and adds snapshot
- adds migrations for geospatial index and updating this data for orgs

## Asana ticket: https://app.asana.com/0/1132189118126148/1198892993658136/f

## PR Checklist




## How to Test

Testing is going to be a bit manual for this one.

1. navigate to catalog homepage
2. Open network inspector in dev tools
3. Perform a location search
4. Compare the "distance" field for search results in network tab. Verify that orgs are listed in order of shortest distance from coordinates
